### PR TITLE
Fixed references to Ellipse, free(), and load() in Asset System reference

### PR DESCRIPTION
**Pull request by tran-dy**
Fixes for issue #607 

Updated the reference to Ellipse and load().
Added the free() method to the Asset reference page to fix the free() hyperlink.

### CODE REVIEW

**Review by pathunstrom — CHANGES_REQUESTED** [review]
This looks fine, we'll skip the system reference itself since we haven't added those docs.

We do need your entry in Contributors.md. Once that's done we can merge!

**Comment by tran-dy** [comment]
Sounds good, contributors file updated!

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -48,4 +48,5 @@ Halle Jones|HJones@aliacy.com||
 [Mateus Denucci Garcia Seabra Resende](https://github.com/MateusDenucci) | |
 [Victor Hart](https://github.com/vicohart) | [vicohart@gmail.com](vicohart@gmail.com) |
 [nilamo](https://github.com/nilamo) | [7nilamo@gmail.com](7nilamo@gmail.com) | 
-[Abhijeet](https://github.com/abhijeetgupto) |[abhigupta7b@gmail.com](abhigupta7b@gmail.com) | [@abhijeetgupto](https://twitter.com/abhijeetgupto)|
\ No newline at end of file
+[Abhijeet](https://github.com/abhijeetgupto) |[abhigupta7b@gmail.com](abhigupta7b@gmail.com) | [@abhijeetgupto](https://twitter.com/abhijeetgupto)|
+[Andy Tran](https://github.com/tran-dy) |
```
Comment (pathunstrom) [inline CONTRIBUTORS.md]:
```suggestion
[Andy Tran](https://github.com/tran-dy) | | |
```

Quick suggestion so the table doesn't break. Thanks!

**Review by pathunstrom — CHANGES_REQUESTED** [review]
Tiny formatting change. Added as suggestion feel free to just press the button to accept it.

**Review by pathunstrom — APPROVED** [review]
Looks good. Thank you for your contribution!

bors r+

**Comment by pathunstrom** [comment]
Don't worry about the conflict, I'll clean it up as part of our merge process.

**Comment by pathunstrom** [comment]
bors retry

**Pull request merged**